meta-lmp-base/recipes-security/optee/optee-os_git.bb

```diff
@@ -19,6 +19,7 @@ SRC_URI_append_imx = " \
     file://0001-imx-huk-imx7-and-imx7ulp-caam-clock-support.patch \
     file://0001-plat-imx-configure-the-SHMEM-section.patch \
     file://0001-imx-caam-check-if-otpmk-can-be-used-when-producing-h.patch \
+    file://0001-config-overlay-reserved-memory-size-addr-cells.patch \
```

Comment:
This is not specific to imx, so fine to move it to the generic SRC_URI line.

Reply:
@ldts ^

Reply:
updated.

Reply:
not sure why I dont get these notifications...anyway sorry about the delay!